rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```
@@ -3198,4 +3250,24 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
         ScheduledExecution.findByUuidAndProject(uuid, project)
     }
 
+    def isProjectExecutionEnabled(String project){
+        def fwProject = frameworkService.getFrameworkProject(project)
+        def disableEx = fwProject.getProjectProperties().get("project.disable.executions")
+        ((!disableEx)||disableEx.toLowerCase()!='true')
+    }
+
+    def isProjectScheduledEnabled(String project){
+        def fwProject = frameworkService.getFrameworkProject(project)
+        def disableSe = fwProject.getProjectProperties().get("project.disable.schedule")
+        ((!disableSe)||disableSe.toLowerCase()!='true')
+    }
+
+    def shouldScheduleInThisProject(String project){
+        try {
+            return isProjectExecutionEnabled(project) && isProjectScheduledEnabled(project)
+        }catch (Exception e){
```

why is this silently catching an exception?